codalab/model/bundle_model.py

```diff
@@ -1491,7 +1492,7 @@ def make_condition(field, value):
 
         # Sort
         if sort_key[0] is not None:
-            query = query.order_by(sort_key[0])
+            query = query.order_by(sort_key[0], sort_key[1])
```

Comment:
This would be cleaner and would scale if we ever added more items to the sort_key:

```suggestion
            query = query.order_by(*sort_key)
```

Reply:
Maybe better to comment what exactly is in the sort key?